### DISCUSSION TITLE
Software needed to run Quarto

### DISCUSSION THREAD

**Discussion by BradySullivan1** [quarto-dev/quarto-web#1163] [General]
Hello, I am very new to programming and am trying to run Quarto. I was following the introduction to Quarto on the website, using it alongside the Jupyter notebook, but when I try to execute,  quarto render hello.ipynb --to html, I get the error ERROR: No valid input files passed to render. I went through other posts to try and trouble shoot it and found out I needed R to run it so I installed r and Rstudio then Rmarkdown and knitr. After all that I tried it again and got the same error.; Any ideas of what else I can do to fix this? Thank you.

**Comment by cscheid** [comment]
You probably ran that command from a different folder than that where the `hello.ipynb` exists.

**Reply by BradySullivan1** [reply]
Thank you, I had it in a folder that was inside the folder that ran the command. When I moved it out it worked. Any idea on how I could get it to work while keeping it in the folder?
